activemq-all/pom.xml

```diff
@@ -14,7 +14,7 @@
   <parent>
     <groupId>org.apache.activemq</groupId>
     <artifactId>activemq-parent</artifactId>
-    <version>5.16.4-SNAPSHOT</version>
+    <version>5.16.5-SNAPSHOT</version>
```

Comment:
PR needs rebased to remove the release commits deleted from the branch.

Reply:
Agree, that's not up to date with the branch state.